terraform-infra-approvals/tax-tribunals-shared-infrastructure.json

```
@@ -0,0 +1,8 @@
+{
+  "resources": [
+    {"type": "azurerm_subnet"}
+  ],
+  "module_calls": [
+    {"source":  "git@github.com:hmcts/tax-tribunals-shared-infrastructure?ref=master"}
+  ]
```

not needed it's loaded based on the filename just do:

```suggestion
  "module_calls": []
```